src/components/molecules/StartTable/StartTable.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import classNames from "classnames";
+import { useAsyncFn } from "react-use";
+
+import {
+  TABLE_COLUMN_WIDTH,
+  TABLE_COLUMN_HEIGHT,
+  TABLE_COLUMN_INDENT,
+} from "settings";
+
+import { updateVenueTable } from "api/table";
+
+import { Table } from "types/Table";
+import { VenueTemplate } from "types/venues";
+
+import { currentVenueSelector } from "utils/selectors";
+
+import { useSelector } from "hooks/useSelector";
+
+import "./StartTable.scss";
+
+export interface StartTablePropsType {
+  tables: Table[];
+  newTable: Table;
+}
+
+export const StartTable: React.FC<StartTablePropsType> = ({
+  tables,
+  newTable,
+}) => {
+  const venue = useSelector(currentVenueSelector);
+
+  const [table] = tables;
```

Comment:
This will get the first table in the array and always use it to calculate `width` and `height`, is this the desired flow?

Reply:
- `src/components/templates/Jazzbar/JazzTab/constants.ts`
- `src/components/templates/ConversationSpace/constants.ts`

Looks like all tables has the same columns/rows configuration, that's why I'm using just the first table to calculate dimensions for the "Start a table" button